ibis/tests/all/test_window.py

```diff
@@ -144,7 +153,7 @@ def test_window(backend, alltypes, df, con, result_fn, expected_fn):
         )
 
     expr = alltypes.mutate(
-        value=result_fn(
+        val=result_fn(
```

Comment:
here I changed it to `val` because `value` raises some error with `MapD`

Reply:
oh interesting, is that a known issue or expected behavior?